packages/eslint-plugin-template/tests/rules/interactive-supports-focus/cases.ts

```diff
@@ -0,0 +1,254 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import type { MessageIds } from '../../../src/rules/interactive-supports-focus';
+
+const messageId: MessageIds = 'interactiveSupportsFocus';
+
+export const valid = [
+  // no interactive outputs
+  { code: '<div></div>' },
+
```

Comment:
In the description, you mentioned that an element is valid if it is disabled. So what if a user ends up with something like `<a disabled>Click me</a>`? Is this considered valid? Do we want a test for it?

Reply:
That's a good catch. I will add a valid test case for the disabled attribute.
The `disabled` attribute is only valid on `<button>`, `<fieldset>`, `<input>`, `<optgroup>`, `<option>`, `<select>`, and `<textarea>` elements. All of those elements pass the IsInteractiveElement test and would receive focus by default when not disabled, but adding disabled to those elements would remove them from the focus ring.